### PR TITLE
Added tests for Refresh button on prepared transactions page

### PR DESCRIPTION
**Pull request by honza-kasik**

### CODE REVIEW

**Comment by honza-kasik** [comment]
@zebrik Will you please review his PR?

File: transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java [hunk 1]
```diff
@@ -102,18 +106,39 @@ public void before() {
...
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[1], PerformedAction.COMMIT));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[2], PerformedAction.COMMIT));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[3], PerformedAction.ROLLBACK));
         add(new PreparedTransactionTestCell(TRANSACTIONS_XID[4], PerformedAction.COMMIT));
-        add(new PreparedTransactionTestCell(TRANSACTIONS_XID[5], PerformedAction.ROLLBACK));
-        add(new PreparedTransactionTestCell(TRANSACTIONS_XID[6], PerformedAction.COMMIT));
         }
     };
 
+    private static final int[] REFRESH_TEST_INDEXES = {5, 6};
```
Comment (zebrik) [inline transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java]: Actually I don't understand the need for `REFRESH_TEST_INDEXES` constant array. IMHO this is just making code less readable. You don't have something like `GUI_TEST_INDEXES` (and not having such variable is ok IMO)

File: transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java [hunk 2]
```diff
@@ -102,18 +106,39 @@ public void before() {
...
+                operations.invoke("commit-prepared-transaction",
+                DEFAULT_MESSAGING_SERVER_ADDRESS, Values.of("transaction-as-base-64", testedXIDs[1])).booleanValue());
+
+        page.refresh();
+
+        Assert.assertNull("Committed transaction should not be present in table after refresh",
+                page.getResourceManager().getResourceTable().getRowByText(0, testedXIDs[0]));
+        Assert.assertNull("Committed transaction should not be present in table after refresh",
+                page.getResourceManager().getResourceTable().getRowByText(0, testedXIDs[1]));
+    }
```
Comment (zebrik) [inline transactions/src/test/java/org/jboss/hal/testsuite/test/transactions/TransactionCommitAndRollbackTestCase.java]: I suggest to add some assert that some prepared transactions remain in the table after the refresh.

**Review by zebrik — COMMENTED** [review]
Hi, just minor complaints ;-)

**Comment by honza-kasik** [comment]
Thanks! I worked in suggestions and improved test a little bit.

**Comment by zebrik** [comment]
@honza-kasik LGTM now, please squash and I will merge this.

**Comment by honza-kasik** [comment]
Thanks! Squashed and ready to be merged.

**Comment by zebrik** [comment]
Thank you. Merging...

**Pull request merged**